### PR TITLE
[DVT-121] add github actions build workflow

### PR DESCRIPTION
**Pull request by rebelArtists**
**jira ticket:** 
https://polygon.atlassian.net/browse/DVT-121

**purpose:**
- provide automatic build / release pipeline for avail explorer artifacts, instead of having to build on remote hosts (devnets)
- all output binaries are compressed for fastest remote download

**tests:**
smoke tests, successful build action workflow run: https://github.com/maticnetwork/avail-apps/actions/runs/3161209787

### CODE REVIEW

File: .github/workflows/releaser.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+             path: /home/runner/work/avail-apps/avail-apps/packages/apps/avail-apps-linux-amd64.tar.gz
+             
+  build_linux_aarch64:
+    runs-on: ubuntu-latest
+    steps:
+         - uses: actions/checkout@v2
+         - name: build avail explorer via yarn
+           shell: bash
+           run: |
+            npm_config_target_arch=arm64 npm_config_target_platform=linux npm_config_target_libc=glibc yarn install
```
Comment (rebelArtists) [inline .github/workflows/releaser.yml:36]: this may need some tweaks as we test with our arm devnets

File: .github/workflows/releaser.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    runs-on: ubuntu-latest
+    steps:
+         - uses: actions/checkout@v2
+         - name: build avail explorer via yarn
+           shell: bash
+           run: |
+            yarn install
+            yarn run build:www
+            pushd /home/runner/work/avail-apps/avail-apps/packages/apps/
+            tar czf avail-apps-linux-amd64.tar.gz build/
```
Comment (praetoriansentry) [inline .github/workflows/releaser.yml:22]: This is just a static website, so I think you can just name it like... `avail-explorer.tar.gz`

File: .github/workflows/releaser.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            popd
+         - uses: actions/upload-artifact@v2
+           with:
+             name: avail-apps-linux-amd64-build
+             path: /home/runner/work/avail-apps/avail-apps/packages/apps/avail-apps-linux-amd64.tar.gz
+             
+  build_linux_aarch64:
```
Comment (praetoriansentry) [inline .github/workflows/releaser.yml:29]: I think you can drop this step. We should only need one generic zip.

File: .github/workflows/releaser.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
+name: Releaser
+on:
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'v*.*.*'
```
Comment (praetoriansentry) [inline .github/workflows/releaser.yml:7]: idk if this is a big deal or not but I switched to just `*` we might want to standardize. The reason was that this globe doens't support a full semvar that's like `v1.2.3-deadbeef+alpha`

**Review by praetoriansentry — CHANGES_REQUESTED** [review]

File: .github/workflows/releaser.yml (hunk 4 above)
Comment (rebelArtists) [reply .github/workflows/releaser.yml]: sg

**Review by praetoriansentry — APPROVED** [review]
lgtm

**Pull request merged**